docs/api.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
Huh?

Reply:
I've spend time trying to figure out which merge was wrong, but couldn't replicate it.
I manually checked the code and I came up with the following PR that need to be merged to fix this:

- https://github.com/asyncapi/parser-js/pull/802
- https://github.com/asyncapi/parser-js/pull/804